docs/source/custom_method_order.py

```diff
@@ -0,0 +1,165 @@
+from inspect import getmembers, ismethod
+import manim as mn
+
+custom_method_order = {
```

Comment:
Am I right, that this dict is written manually?
If this is the case, what will happen to the docs if one of these methods will be removed for a future release?
Should this file then also be maintained manually?

Reply:
Yes, this is the main sticking point of the current PR. However, if you see the bottom of the file, I added some code to guarantee that all methods added in the dict are actual methods of that class, and also that all methods not included in this dictionary are included in a default "Misc. methods" section. 

So: if a method included in this dict is removed, this file should raise an error. If a new method is added to the class but not to the dictionary, it will still show up in the docs in the "Misc" section.

Reply:
Ok, sounds good!
Can we document this behaviour maybe here?
https://docs.manim.community/en/stable/contributing.html#further-useful-guidelines

Reply:
Yes, please document this mechanism in a new point in the "Further useful guidelines" -- otherwise, I also think that this is good to go; excellent idea!